config/routes.rb

```diff
@@ -19,5 +23,7 @@
 
   resources :reviews, only: [:edit, :update, :destroy]
 
+  resources :favorites, only: [:index, :new, :create, :destroy]
```

Comment:
Sorry for not catching this earlier, but you only have an `index` action defined in your Favorites controller and so you should only have that route allowed here!